src/main/java/com/ays/common/model/entity/BaseEntity.java

```diff
@@ -53,6 +54,7 @@ public void prePersist() {
     public void preUpdate() {
         this.updatedUser = Optional.ofNullable(SecurityContextHolder.getContext().getAuthentication())
                 .map(Authentication::getPrincipal)
+                .filter(user -> !"anonymousUser".equals(user))
```

Comment:
Olabiliyorsa şöyle düzeltebilir miyiz.

`.filter(user -> user instanceof Jwt && !"anonymousUser".equals(user))`

Reply:
Aslında ikisi birlikte olmuyor, buradaki filter'daki amaç `anonymousUser` içeren bir String object geldiğinde token olmadan istek atılabilen bir endpoint olduğunu anladığımız için onu göz ardı edip, default olarak `AYS` yazmasını sağlıyoruz.